pertpy/tools/_metadata/_cell_line.py

```diff
@@ -108,6 +108,10 @@ def __init__(self):
                 zip_file.extract("rnaseq_read_count_20220624.csv", path=settings.cachedir)
 
         self.bulk_rna_sanger = pd.read_csv(bulk_rna_sanger_file_path, skiprows=[2, 3], header=[0, 1], index_col=[0, 1])
+        # remove unnecessary space in read count values
```

Comment:
Could you please add more context here? As we discussed, what is the issue? (random white space and we don't know whether it should be there or not etc). If possible, always document WHY you're doing something